frontend/src/components/details/SelectVariableComponent.jsx

```diff
@@ -16,23 +16,76 @@ class SelectVariableComponent extends Component {
   */
     state = {
         selectedVariable: 'variable 1',
+        secondVariable: 'variable 2',
         variables: [
-            { id: '1', country: 'variable 1' },
-            { id: '2', country: 'variable 2' },
-            { id: '3', country: 'variable 3' }
+            { id: '1', name: 'variable 1' },
+            { id: '2', name: 'variable 2' },
+            { id: '3', name: 'variable 3' }
         ]
     };
 
     handleSubmit = () => {
         alert('The choosen variable is: ' + this.state.selectedVariable);
     };
 
-    onDropDownItemSelectedHandler = (country) => {
-        const selectedVariable = country;
+    onDropDownItemSelectedHandler = (name) => {
+        const selectedVariable = name;
         this.setState({ selectedVariable });
     };
 
+    onSecondDropDownSelectedHandler = (name) => {
+        const secondVariable = name;
+        this.setState({ secondVariable });
+    };
+
     render() {
+        /*
+        if the loadComparePage state from its parrent (the detail Component) 
+        is set to true, here an extra drop down for the second variable
+         should be rendered 
+        */
+        if (this.props.loadComparePage) {
+            return (
+                <div>
+                    Pick your desire variable:
+                    <div className='w3-dropdown-hover w3-margin-left w3-margin-right w3-margin-top'>
```

Comment:
@ScoutAtlas and I were wondering what these classNames do, can you please explain? ._.

Reply:
sure, do you remember that I said I use a CSS framework called "W3.css". 
the attributions of the CSS frameworks should be defined there. 
for example, here it means that this is a type of drop-down class and the other 3 defines obviously the margin.

Reply:
thank you for clarifying ☺️